<launch>
  <!-- text-davinci-003 was deprecated on January 4, 2024,
    so now use GPT-3.5-turbo-instruct. -->
  <!-- <arg name="model_name" default="text-davinci-003"/> -->
  <arg name="model_name" default="gpt-3.5-turbo-1106" />
  <arg name="gen_file" default="bhv" />
  <arg name="mission_name" default="test" />
  <arg name="count" default="1" />

  <node name="behavior_tree_generation" pkg="behavior_tree_generation" type="behavior_tree_generation_moos.py" output="screen">
    <param name="mission_name" value="$(arg mission_name)" />
    <param name="gen_file" value="$(arg gen_file)" />
    <param name="model_name" value="$(arg model_name)" />
    <param name="count" value="$(arg count)" />
  </node>

</launch>